Diagnostic/diagnostic.py

```diff
@@ -1166,6 +1175,51 @@ def get_deployment_id():
 
     return identity
 
+class Watcher():
+    lastModTime = os.path.getmtime('/etc/fstab')
+    logger = None
+
+    def __init__(self, errorStream, outputStream):
+        self.logger = logging.getLogger(__name__)
+        self.logger.setLevel(logging.INFO)
+
+        ch = logging.StreamHandler(errorStream)
+        ch.setLevel(logging.WARNING)
+        self.logger.addHandler(ch)
+        ch = logging.StreamHandler(outputStream)
+        ch.setLevel(logging.INFO)
+        self.logger.addHandler(ch)
+
+    def handle_fstab(self, ignoremtime=False):
+        tryMount = False
+        if ignoremtime == True:
+            tryMount = True
+        else:
+            currentModTime = os.path.getmtime('/etc/fstab')
+            currentModDateTime = datetime.datetime.fromtimestamp(currentModTime)
+            
+            if (currentModTime != self.lastModTime and 
+                currentModDateTime < currentModDateTime + 
```

Comment:
added a comment.  I'm trying to only check with mount after at least 1 minute has elapsed since the modification